packages/turf-transform-rotate/index.d.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference types="geojson" />
+import {Point} from '@turf/helpers'
+
+export type GeometryObject = GeoJSON.GeometryObject;
+export type Feature<Geom extends GeometryObject> = GeoJSON.Feature<Geom>;
```

Comment:
Since this modules are single export methods, you can only have 1 dedicated export, can't have multiple exports.
```
index.d.ts(15,1): error TS2309: An export assignment cannot be used in a module with
 other exported elements.
```